confluent-kafka-plugins/src/main/java/io/cdap/plugin/confluent/streaming/source/ConfluentStreamingSourceConfig.java

```diff
@@ -306,7 +311,7 @@ public Integer getMaxRatePerPartition() {
   @Nullable
   public Schema getSchema() {
     try {
-      return Strings.isNullOrEmpty(schema) ? null : Schema.parseJson(schema);
+      return Strings.isNullOrEmpty(schema) ? DEFAULT_SCHEMA : Schema.parseJson(schema);
```

Comment:
This is not consistent with Kafka streaming plugin. There we return null if schema is empty.

Reply:
The bug exists in Kafka Streaming Plugin as well. On clearing the schema from the Actions menu, when the user clicks on Validate button, this error is thrown.
_*Required property 'schema' has no value._

<img width="1735" alt="Kafka Plugins Schema Validation Error" src="https://user-images.githubusercontent.com/89522746/228505359-9beb82aa-3b01-4d7a-8e2b-835596e4d6c1.png">

Reply:
Can you create a bug for the Kafka plugin as well?

Reply:
Yes, [PLUGIN-1565](https://cdap.atlassian.net/browse/PLUGIN-1565).